joomla/joomla-cms
# [com_fields] Support clearing the custom fields data

Pull Request for Issue #12812.

### Summary of Changes
Supports to clear the data.

### Testing Instructions
1. Create a custom field with at least two options and set the "Multiple" parameter to "Yes"
2. Open an item and see the empty box for the field. Select one or more options and save.
3. Open the item again and see the selected options. That should work fine.
4. Now try to remove all selected options, making the box empty again and save the item.
5. Check the item item again

### Expected result
The box should stay empty

### Actual result
The box keeps the last saved options, making it impossible to reset the field to "nothing selected"

### Note
Please test around with other field types adding and removing data, to make sure all is working as expected.
